### ISSUE TITLE
logstash-input-rss issue with proxy 

### ISSUE THREAD

**Issue by DBARUNNER** [logstash-plugins/logstash-input-rss#23]
I am using logstash 5.1.2 when i run the input-rss plugin with out proxy it works fine for me but when i run it with proxy it give me error any one could hel me

**Comment by kunisen** [comment]
Any updates here? It looks like we have the same issue in 5.6.

**Comment by DBARUNNER** [comment]
No, i totally changed my field now im not working with logstash
